src/xcm_config.rs

```diff
@@ -0,0 +1,91 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use codec::{Decode, Encode};
+
+use cumulus_primitives_core::ParaId;
+use frame_support::RuntimeDebug;
+use sp_std::prelude::*;
+
+#[derive(Encode, Decode, RuntimeDebug)]
+pub enum SystemCall {
+    #[codec(index = 8)]
+    RemarkWithEvent(Vec<u8>),
+}
+
+#[derive(Encode, Decode, RuntimeDebug)]
+pub enum AutomationTimeCall<T: frame_system::Config> {
+    #[codec(index = 0)]
+    ScheduleNotifyTask(Vec<u8>, Vec<u64>, Vec<u8>),
+    #[codec(index = 2)]
+    ScheduleXcmpTask(Vec<u8>, Vec<u64>, ParaId, Vec<u8>, u64),
```

Comment:
only this extrinsic is setup to accept `OriginKind::Native`. All the others will fail due to a bad origin. We either need to send those with the sovereign origin type or not support them in the crate. I vote for not supporting them as they add no real value.